web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v3110/migrate-oracle.sql

```diff
@@ -11,3 +11,5 @@ ALTER TABLE Validation ALTER COLUMN valType TYPE varchar(128);
 
 -- New setting for server timezone
 INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('system/server/timeZone', '', 0, 260, 'n');
+
+ALTER TABLE usersearch ALTER COLUMN url TYPE text;
```

Comment:
According to the following
https://asktom.oracle.com/pls/asktom/f?p=100:11:0::::P11_QUESTION_ID:1770086700346491686

It looks like it would be multiple commands for oracle and they would be the following.

ALTER TABLE usersearch ADD ( temp clob );
ALTER TABLE usersearch SET temp=url , url =null;
ALTER TABLE usersearch DROP column url ;
ALTER TABLE usersearch RENAME COLUMN temp to url ;